quick_install.sh

```diff
@@ -782,8 +782,41 @@ function __bliman_load_export_vars() {
 	genesis_file_path=$1
 	sed -i '/^$/d' "$genesis_file_path" # Delete empty lines
 	genesis_data=$(<"$genesis_file_path")
+	
 	source_file="$BLIMAN_DIR/tmp/source.sh"
 	__bliman_convert_yaml_to_sh "$genesis_data" "$source_file"
 }
 
-__bliman_quick_install "$1"
+#### MAIN STARTS HERE
+opts=()
+args=()
+while [[ -n "$1" ]]; do
+  case "$1" in
+        --genesis_path | --force)         
+           opts=("${opts[@]}" "$1")
+	   ;; ## genesis file path on local system
+        *)          
+           args=("${args[@]}" "$1")
+	   ;; ## command | genesis_path
+  esac
+  shift
+done
+[[ -z $command ]] && command="${args[0]}"
+case $command in
+     install)
```

Comment:
What is the use of this code?

Reply:
This code is added to create install, uninstall and update features for the bliman on user's machine.